### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Stasandrushchak**
- [DEMO LINK](https://Stasandrushchak.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Stasandrushchak.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,13 +10,118 @@
...
+        <ul class="u-list">
+          <li class="nav_list_item">
+            <a
+              href="./index.html"
+              class="nav_link is-active"
+            >
+              apple
+              <img
+                src="./images/Rectangle.png"
+                alt="is_active"
```
Comment (polosanya) [inline src/index.html:62]: It is not correct to add underline as an image. What if another link becomes active? It should have underline too.
Consider implementing it with after/before pseudoelement
https://www.w3schools.com/css/css_pseudo_elements.asp

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
+  padding: 0;
+}
+
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-bold {
```
Comment (polosanya) [inline src/style.css:21]: There are no such selectors(classes) as .roboto-regular, .roboto-bold etc..

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,112 @@
...
+  text-align: left;
+}
+
+.is-active::after {
+  position: relative;
+  background-image: url(./images/Rectangle.png);
+  bottom: 4;
+}
+
+.video {
```
Comment (polosanya) [inline src/style.css:110]: Don't hardcode the name of the class according to the link name. What if it changes?

**Review by polosanya — CHANGES_REQUESTED** [review]
Nice start! Let's try to pass all tests
Pay attention to the design. Header should have box-shadow property
image

File: src/index.html (hunk 1 above)
Comment (Stasandrushchak) [reply src/index.html:62]: Thank you very much for advice, and link)

File: src/style.css (hunk 3 above)
Comment (Stasandrushchak) [reply src/style.css]: Can inspect my new code?

**Pull request closed without merging**